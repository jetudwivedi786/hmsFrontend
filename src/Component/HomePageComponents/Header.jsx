import React from 'react'
import Logo from "./logo2.png"

const Header = () => {
    return (
        <>
            <div style={{ backgroundColor: "#912e2e" }}>
                <h1><img src={Logo} style={{ width: 140, height: 140, borderRadius: 70 }} />
                    <span style={{ paddingLeft: "10%", color: "white", fontWeight: "bold", fontFamily: 'Sansita Swashed', fontSize: "50px" }}> Hotel Management System</span></h1>
            </div>
        </>
    )
}

export default Header