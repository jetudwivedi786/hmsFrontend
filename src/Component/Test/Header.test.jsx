import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../HomePageComponents/Header.jsx';

it('renders correctly', () => {

    const tree = renderer

        .create(<Header />)

        .toJSON();

    console.log(tree)

    expect(tree).toMatchSnapshot();

});