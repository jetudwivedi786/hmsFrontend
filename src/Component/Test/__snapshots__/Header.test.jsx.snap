// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#912e2e",
    }
  }
>
  <h1>
    <img
      src="logo2.png"
      style={
        Object {
          "borderRadius": 70,
          "height": 140,
          "width": 140,
        }
      }
    />
    <span
      style={
        Object {
          "color": "white",
          "fontFamily": "Sansita Swashed",
          "fontSize": "50px",
          "fontWeight": "bold",
          "paddingLeft": "10%",
        }
      }
    >
       Hotel Management System
    </span>
  </h1>
</div>
`;
